src/gpu/nvidia/sycl_cuda_engine.cpp

```diff
@@ -223,6 +224,14 @@ constexpr dnnl::impl::impl_list_item_t sycl_cuda_impl_list[] = {
         INSTANCE(cudnn_lrn_fwd_t)
         INSTANCE(cudnn_lrn_bwd_t)
 
+        INSTANCE(sycl::ref_sycl_lrn_fwd_t<data_type::f32>)
```

Comment:
Please refactor the code to avoid instantiating the primitive for each data type. You can use eltwise primitive for reference.

Reply:
Done. Please check https://github.com/oneapi-src/oneDNN/pull/1605